src/Workspaces/SharedUtilitiesAndExtensions/Workspace/VisualBasic/Extensions/ContextQuery/VisualBasicSyntaxContext.vb

```diff
@@ -55,18 +55,20 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions.ContextQuery
             position As Integer,
             leftToken As SyntaxToken,
             targetToken As SyntaxToken,
+            containingTypeDeclaration As SyntaxNode,
             isAttributeNameContext As Boolean,
             isAwaitKeywordContext As Boolean,
-            isCustomEventContext As Boolean,
+            isBaseClassContext As Boolean,
+            isBaseInterfaceContext As Boolean,
+            IsCustomEventContext As Boolean,
             isEnumTypeMemberAccessContext As Boolean,
             isAnyExpressionContext As Boolean,
             isGenericConstraintContext As Boolean,
             isGlobalStatementContext As Boolean,
-            isOnArgumentListBracketOrComma As Boolean,
+            IsOnArgumentListBracketOrComma As Boolean,
```

Comment:
why are some parameters capitalized but others are not?  seems wrong.

Reply:
Probably some codeaction result, It doesn't seem like I did it consciously myself